### PR TITLE
chore: Update README to correct branch pairing instructions

### PR DESCRIPTION
**Pull request by rajivnathan**
Updates the README to reflect the changes made in https://github.com/codeready-toolchain/toolchain-e2e/pull/61. It also explains the reason for the behaviour.

### CODE REVIEW

**Comment by alexeykazakov** [comment]
Thanks for updating the docs.

File: README.adoc [hunk 1]
```diff
@@ -82,12 +83,12 @@ If you are still confused by the different e2e/operator location, execution and
...
 *** run `make test-e2e-member-local`
-** *You are working in both repos `toolchain-e2e` and `host-operator`, so you need to run e2e tests against your current code located in `../host-operator` directory:*
+** *You are working in both repos `toolchain-e2e` and `host-operator` with a matching branch name, so you need to run e2e tests against your current code located in `../host-operator` directory:*
 *** run `make test-e2e-host-local`
-** *You are working in all three repos `toolchain-e2e`, `host-operator` and `member-operator`, so you need to run e2e tests against your current code located in both directories `../host-operator` and `../member-operator`:*
-*** run `make test-e2e-local`
+** *You are working in all three repos `toolchain-e2e`, `host-operator` and `member-operator`*
+*** This is prohibited and will result in an error like `ERROR WHILE TRYING TO PAIR PRs` in the CI build. See the explanation in <<End-to-End Tests>> section. You should make functional changes to the operator repositories independently.
```
[the remark this answers is not in the document]
Comment (rajivnathan) [reply README.adoc:91]: Good catch, I've updated it now. Let me know if that resolves your concern

**Pull request merged**